wazimap_ng/datasets/models/dataset.py

```diff
@@ -22,6 +22,16 @@ def __str__(self):
     class Meta:
         ordering = ["id"]
 
+meta_data_license = (("creative commons", "Creative Commons"), ("other", "Other"))
```

Comment:
I don't think this should be a hardcoded field - probably better to create a database model